AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/test/Fixtures.dfy

```diff
@@ -127,8 +129,23 @@ module Fixtures {
 
   // Creation of this particular illegal Branch Key is detailed here:
   // `git rev-parse --show-toplevel`/cfn/lyingBranchKeyCreation.md
-  const lyingBranchKeyId := "kms-arn-attribute-is-lying"
-  const lyingBranchKeyDecryptOnlyVersion := "129c5c87-308a-41c9-8b9d-a27f66e915f4"
+  const hierarchyV1InvalidKmsArnId := "kms-arn-attribute-is-lying"
+  const hierarchyV1InvalidKmsArnVersion := "129c5c87-308a-41c9-8b9d-a27f66e915f4"
+  // TODO-HV-2-FF : Document creation of lying branch keys
+  const hierarchyV2InvalidKmsArnId := "DO-NOT-DELETE-test-hv2-get-key-wrong-kms-arn"
```

Comment:
Non-blocking: We could append DONOTDELETE to all these variable so that the next developer will be extra careful even when using variable name.

We are not doing this in any variable so I am also fine with what is here.